services/metadata/twoplaysamonth.js

```diff
@@ -1,7 +1,8 @@
 export const Config = {
   name: "twoplaysamonth",
   display: "#2PlaysAMonth",
-  description: "#2PlaysAMonth is an event by ReactPlay for the ReactJs community to learn, build, share in public. Join us.",
+  description:

```

Comment:
Can we ignore this new line?

Reply:
Sure, I'll fix that. It happened due to my VS Code formatter

Reply:
I removed the new line